tests/integration/query/one_to_many/with_count_test.go

```diff
@@ -118,3 +118,66 @@ func TestQueryOneToManyWithCount(t *testing.T) {
 		executeTestCase(t, test)
 	}
 }
+
+// This test documents the behavior of aggregate alias targeting which is not yet implemented.
+// https://github.com/sourcenetwork/defradb/issues/3195
+func TestQueryOneToMany_WithCountAliasFilter_Succeeds(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "One-to-many relation query from many side with count",
+		Actions: []any{
+			testUtils.CreateDoc{
+				CollectionID: 0,
+				Doc: `{
+					"name": "Painted House",
+					"rating": 4.9,
+					"author_id": "bae-e1ea288f-09fa-55fa-b0b5-0ac8941ea35b"
```

Comment:
todo: Please use `DocIndex` for these IDs instead of strings, it reduces the maintenance and readability overhead quite a bit IMO, it will also force the test to create the docs in a 'correct' order :)

Reply:
done